src/wakefs/fuse.cpp

```diff
@@ -217,9 +227,35 @@ bool run_in_fuse(fuse_args &args, int &status, std::string &result_json) {
   (void)close(STDOUT_FILENO);
   (void)close(STDERR_FILENO);
 
-  do waitpid(pid, &status, 0);
-  while (WIFSTOPPED(status));
```

Comment:
I'm not sure what the `WIFSTOPPED` is supposed to be doing here, so deleting it may be problematic. @JakeSiFive @mmjconolly

Reply:
I think its saying that if the child was signaled, just keep waiting on it anyway

Reply:
I suspect there's actully a bug here where if a child is stopped via SIGSTOP, and then continued wakebox will treat the process as having been terminated when it hasn't. I think the correct condition is `!WIFEXITED(status)`